analysis/src/CompletionBackEnd.ml

```diff
@@ -1135,6 +1135,32 @@ and getCompletionsForContextPath ~debug ~full ~opens ~rawOpens ~pos ~env ~exact
                      | _ -> false
                    else true)
         in
+
+        (* TODO(config-complete-builtins) This config and config resolution should come from `package` instead. *)
+        let pretendPackageBuiltinsConfig =
+          Misc.StringMap.of_list
+            [("array", ["ArrayUtils"]); ("Fastify.t", ["FastifyExt"])]
```

Comment:
This is an array of identifiers because you could configure several modules to complete from.

The format should be `Path.To.Module`. These examples are all single module paths, but could've just as easily been for example `Utils.Array` instead of `ArrayUtils`. Might be good adding test cases for that as well.